angel_system/ptg_eval/activity_classification/compute_scores.py

```diff
@@ -2,23 +2,23 @@
 from sklearn.metrics import average_precision_score
 import numpy as np
 
-log = logging.getLogger("ptg_eval")
+log = logging.getLogger("ptg_eval_activity")
 
 
 class EvalMetrics():
-    def __init__(self, labels, gt_true_mask, dets_per_valid_time_w, output_dir=''):
+    def __init__(self, labels, gt_true_mask, window_class_scores, output_dir=''):
         """
         :param labels: Array of class labels (str)
         :param gt_true_mask: Matrix of size (number of valid time windows x number classes) where True
             indicates a true class example, False inidcates a false class example. There should only be one
             True value per row
-        :param dets_per_valid_time_w: Matrix of size (number of valid time windows x number classes)
+        :param window_class_scores: Matrix of size (number of valid time windows x number classes)
             filled with the max confidence score per class for any detections in the time window
```

Comment:
"filled with the max confidence" is a decision that happened upstream of this classes' scope. This class just takes scores for windows and gives and overall score. E.g., we might decide at some future time to take the min, median, or mean score in the window, but this class doesn't care about what was done.